router/core/graphql_handler.go

```diff
@@ -205,7 +204,6 @@ func (h *GraphQLHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 		err := h.executor.Resolver.ResolveGraphQLSubscription(ctx, p.Response, writer)
 		requestContext.dataSourceNames = getSubgraphNames(p.Response.Response.DataSources)
-		requestContext.telemetry.AddCustomMetricStringSliceAttr(ContextFieldOperationServices, requestContext.dataSourceNames)
```

Comment:
Where are those fields added if not here?

Reply:
They were already added here. I didn't see that they were used in the `graphql_handler.go`:

https://github.com/wundergraph/cosmo/blob/0c6bd041e88bc6026f837fcb5391b663211a829d/router/core/graphql_prehandler.go#L200-L203